msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/wpj/TestCase1561080.java

```diff
@@ -100,11 +102,14 @@ public void test_1561080() throws Throwable {
         UiObject disable_lost_device_btn2 = UiAutomatorUtils.obtainUiObjectWithText("Disable lost device");
         disable_lost_device_btn2.click();
 
-        //install prod brokerhost app.
+        // Install BrokerHost.apk
         BrokerHost brokerHost = new BrokerHost(BrokerHost.BROKER_HOST_APK_RC);
-        brokerHost.clear();
+        if(brokerHost.isInstalled()){
+            brokerHost.uninstall();
```

Comment:
I think we should probably throw an `AssertionError` here. Existing installation found at this step should be considered a test failure.

Reply:
Will do, good call